Dockerfile

```
@@ -16,7 +16,7 @@
 
 ## Build
 
-FROM docker.io/library/golang:1.21 AS build
+FROM docker.io/library/golang:1.22 AS build
```

Not sure if we want to, but we could pin to the patch version here...

```suggestion
FROM docker.io/library/golang:1.22.5 AS build
```

---

I think it's ok to just use `1.22`. Go is usually pretty stable between patch releases.